apps/redi-talent-pool/src/assets/locales/en/translation.json

```diff
@@ -51,127 +55,122 @@
+            "answer": "The ReDI Talent Pool is accessible to all ReDI job seekers and contains profiles from current and former ReDI students who have various skill-sets. The ReDI community consists of 104 different nationalities, over 60% identify themselves as female and more than 70% have obtained a university degree (on top of the ReDI certification). Yes - we are proud of our diversity!"
           },
           {
-            "question": "How often and for how long should we meet?",
-            "answer": "We recommend a minimum of 6 sessions in a period of about 6 months, 1 hour each time. However, a lot of people meet more often and usually for more than an hour. (1,5 - 2 hours seems to be a good time to get something done). The meetings can be scheduled whenever you want to. We just ask mentors to log all meetings on the ReDI Connect platform."
+            "question": "Do ReDI job seekers have a specific professional background?",
+            "answer": "The short answer is no. ReDI students come from different countries and have different backgrounds. Some ReDI students come to Germany with profound tech knowledge and many have already studied at the University in their home country, others have a different (academic) background and seek to start a career in the German tech industry. All ReDI students have one thing in common though: Their interest in tech and their strong motivation to start a new career in Germany."
           },
           {
-            "question": "How can we structure our mentoring sessions?",
-            "answer": "For your first meeting you can focus on getting to know each other and work on setting goals together. Some students already have clear long-term goals. In these cases, you can think carefully about concrete, realistic goals for the next 6 months that bring you closer to your long-term goals. Other students are not sure yet about how to move forward, in which area to specialize or how to find realistic opportunities for the upcoming years. In those cases, your mentorship goals can be to gain insights into one or a few specialization areas, or to determine the best skills to develop further. For more specific guidance, mentors can reach out and learn from other mentors using the Slack space or the regular mentors’ gatherings. There is also a shared folder on our ReDI Community drive with links to resources and other useful guidance."
+            "question": "Are there any requirements for accessing the ReDI Talent Pool as a company?",
+            "answer": "The ReDI Talent Pool serves as a safe space for the ReDI Community and therefore it is also not open to everybody. To access the ReDI Talent Pool as a company representative your company must be a ReDI Career Partner. You should get in touch with our partnership team (partner@redi-school.org) in order to find out how to become a ReDI Career Partner."
```

Comment:
We can make partner@redi-school.org a link here. Just like you do on the line 129.